packages/server/src/main/resources/application-local.properties

```diff
@@ -92,6 +92,21 @@ spring.data.redis.url=redis://localhost:6379
 ########################################################################################################################
 # TaskRunner configuration
 ########################################################################################################################
+terarium.taskrunner.rabbitmq.default.addresses=amqp://localhost:5672
+terarium.taskrunner.rabbitmq.default.username=terarium
+terarium.taskrunner.rabbitmq.default.password=terarium123
+
+# The following taskrunners will connnect to dev. If you want to run them locally, comment these
+# lines out and then uncomment the lines in `docker-compose-taskrunner.yml`.
+terarium.taskrunner.rabbitmq.equation_extraction.addresses=amqp://rabbitmq.dev.terarium.ai:5672
+terarium.taskrunner.rabbitmq.equation_extraction.username=${terarium.mq-username}
+terarium.taskrunner.rabbitmq.equation_extraction.password=${terarium.mq-password}
+terarium.taskrunner.rabbitmq.table_extraction.addresses=amqp://rabbitmq.dev.terarium.ai:5672
+terarium.taskrunner.rabbitmq.table_extraction.username=${terarium.mq-username}
+terarium.taskrunner.rabbitmq.table_extraction.password=${terarium.mq-password}
+terarium.taskrunner.rabbitmq.text_extraction.addresses=amqp://rabbitmq.dev.terarium.ai:5672
+terarium.taskrunner.rabbitmq.text_extraction.username=${terarium.mq-username}
+terarium.taskrunner.rabbitmq.text_extraction.password=${terarium.mq-password}
```

Comment:
where do you define those `terarium.mq-[username|password]` when running locally?